.editorconfig

```diff
@@ -4,11 +4,38 @@ end_of_line = "lf"
 
 [*.cs]
 indent_style = tab
+indent_size = 4
 csharp_new_line_before_open_brace = types,methods
```

Comment:
Should this be just methods? https://www.mono-project.com/community/contributing/coding-guidelines/ says classes get curly braces on the same line.

Reply:
We haven't been very consistent on this one and both exist. I reviewed also monodevelop version which is using the same `types,methods` and it's also same to the default VS so this setting makes sense to me.